README.md

```diff
@@ -1,33 +1,27 @@
-# [zkSync Checkout dApp](https://checkout.zksync.io) &middot; [zkSync.io](https://zksync.io/)  [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE-MIT) [![GitHub license](https://img.shields.io/badge/license-Apache%202-blue)](./LICENSE-APACHE)
+# [RIF Rollup Checkout dApp](https://checkout.dev.aggregation.rifcomputing.net/) &middot;  [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE-MIT) [![GitHub license](https://img.shields.io/badge/license-Apache%202-blue)](./LICENSE-APACHE)
 
-# zkSync Checkout — trustable permissionless DeFi payment gateway(test)
+# RIF Rollup Checkout — trustable permissionless DeFi payment gateway(test)
 
 zkSync Checkout helps anyone permission-less adopt checkout backed by zkSync, receive payments automatically and benefit from all the advantage of zkSync Rollup: speed of the transaction, times lower cost of a single transaction, simplicity of withdrawal fund to the onchain-wallet.
 
-* **[SDK documentation](https://zksync.io/api/sdk/checkout/)** | [**Changelog**](CHANGELOG.md)
-
-## Deployment
-
-zkSync uses firebase hosting for all it's dApps.
-Resource targets for the zkCheckout are:
+* **[SDK documentation](https://github.com/rsksmart/rif-rollup-js-sdk)** | [**Changelog**](CHANGELOG.md)
 
 ### Available Hosts
 
-* [```prod-mainnet```](https://checkout.zksync.io)
-* [```prod-goerli```](https://checkout-goerli.zksync.io)
-* [```staging```](https://staging-checkout-v1.zksync.dev/link)
+* [```prod-mainnet```](https://checkout.aggregation.rifcomputing.net/)
```

Comment:
I'm thinking about removing every mainnet reference till we actually have a mainnet version. otherwise this information is misleading.

Reply:
ok, I'll remove all references to mainnet then